cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/epilog

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+exec &>>"/var/log/parallelcluster/slurm_prolog_epilog.log"
```

Comment:
I think we want to control the output that goes into the log file and not redirect all stdout and stderr.

As part of this I'd also create a function to take care of logging which also adds INFO or ERROR  to each log entry.